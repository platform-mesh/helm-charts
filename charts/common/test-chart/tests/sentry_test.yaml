suite: sentry
templates:
  - sentry.yaml
values:
  - ../values.yaml
chart:
  namespace: default
  appVersion: 0.1.0
tests:
  - it: test sentry enabled (global)
    set:
      global:
        sentry:
          enabled: true
        externalSecrets:
          enabled: true
    asserts:
      - matchSnapshot: {}
  - it: test sentry enabled (local)
    set:
      sentry:
        enabled: true
      externalSecrets:
        enabled: true
    asserts:
      - matchSnapshot: {}
