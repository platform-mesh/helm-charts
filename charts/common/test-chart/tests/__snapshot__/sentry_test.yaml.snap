test sentry enabled (global):
  1: |
    apiVersion: external-secrets.io/v1beta1
    kind: ExternalSecret
    metadata:
      name: RELEASE-NAME-test-chart-sentry
      namespace: NAMESPACE
    spec:
      data:
        - remoteRef:
            conversionStrategy: Default
            key: sentry/sentry-dsn
            property: RELEASE-NAME-test-chart-sentry
          secretKey: dsn
      refreshInterval: 10m
      secretStoreRef:
        kind: SecretStore
        name: environment-store
      target:
        creationPolicy: Owner
        deletionPolicy: Retain
        name: RELEASE-NAME-test-chart-sentry
test sentry enabled (local):
  1: |
    apiVersion: external-secrets.io/v1beta1
    kind: ExternalSecret
    metadata:
      name: RELEASE-NAME-test-chart-sentry
      namespace: NAMESPACE
    spec:
      data:
        - remoteRef:
            conversionStrategy: Default
            key: sentry/sentry-dsn
            property: RELEASE-NAME-test-chart-sentry
          secretKey: dsn
      refreshInterval: 10m
      secretStoreRef:
        kind: SecretStore
        name: environment-store
      target:
        creationPolicy: Owner
        deletionPolicy: Retain
        name: RELEASE-NAME-test-chart-sentry
